docs/explanation/cryptography.rst

```diff
@@ -0,0 +1,34 @@
+.. _explanation_cryptographic-technology:
+
+Cryptographic technology in Charmcraft
+======================================
+
+Charmcraft uses cryptographic technologies to fetch arbitrary files over the internet,
+communicate with local processes, and store credentials. It does not directly implement
+its own cryptography, but it does depend on external libraries to do so.
+
+Charmcraft is built upon `Craft Application`_ and derives much of its functionality from
```

Comment:
```suggestion
Charmcraft is built on `Craft Application`_ and derives much of its functionality from
```

I wish I hadn't let this word propagate...

Reply:
"We built this city upon rock and roll." -- Starcraft